src/screens/CreatePost/elements/CreatePollContainer/style/CreatePollContainerBaseStyle.js

```diff
@@ -101,14 +100,13 @@
   parentcolumncontainer: {
     display: 'flex',
     flexDirection: 'column',
-    backgroundColor: 'white',
-    paddingVertical: dimen.normalizeDimen(24),
-    borderRadius: 4,
-    paddingHorizontal: dimen.normalizeDimen(12)
+    backgroundColor: COLORS.almostBlack,
+    paddingVertical: dimen.normalizeDimen(20),
+    borderRadius: 9,
+    paddingHorizontal: dimen.normalizeDimen(18)
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in the `parentcolumncontainer` style seem to be for a dark mode feature. If that's the case, consider using a more dynamic approach where styles can be switched based on the current theme (light or dark). This will make your code more maintainable and scalable.
```diff
-    backgroundColor: 'white',
-    paddingVertical: dimen.normalizeDimen(24),
-    borderRadius: 4,
-    paddingHorizontal: dimen.normalizeDimen(12)
+    backgroundColor: COLORS.almostBlack,
+    paddingVertical: dimen.normalizeDimen(20),
+    borderRadius: 9,
+    paddingHorizontal: dimen.normalizeDimen(18)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->